apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -347,6 +349,75 @@ static void tc_libc_signal_psignal(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_libc_signal_sigwait
+* @brief                :tc_libc_signal_sigwait test sigwait function
+* @scenario             :If signo < 0, it return errno by get_errno().
+*                        Else if signo > 0, return signo that signal number in the location referenced by sig.
+* API's covered         :sigwait()
+* Preconditions         :none
+* Preconditions         :none
+* @return               :void
+*/
+
+static int sigwait_receiver(int argc, FAR char *arvg[])
+{
+	sigset_t set = ALL_SIGNAL_SET;
+	int ret_chk, sig;
```

Comment:
int ret_chk;
int sig;

Reply:
please check all of local variables definitions.

Reply:
modify